### ISSUE TITLE
Does this package have a ROS version?

### ISSUE THREAD

**Issue by Kanelankai** [dji-sdk/Tello-Python#13]
Can Tello be developed with ROS?
Thank you!

**Comment by hanker-lu** [comment]
Yes. You can try to connect the  program of tello interaction  to the ROS platform.But firstly you should encapsulate this tello interaction program into a node that can connect to ROS.
